node/src/accountant/payment_adjuster/non_unit_tests/mod.rs

```diff
@@ -647,216 +744,241 @@ fn write_brief_test_summary_into_file(
          {}\n\n\
          Unsuccessful\n\
          Caught by the entry check:............. {}\n\
-         With 'AllAccountsEliminated':.......... {}\n\
+         With 'RecursionDrainedAllAccounts':.... {}\n\
```

Comment:
In my opinion, the word `Eliminated` gives a bigger picture then the word `Drained`. Let's use that instead.